doc/source/whatsnew/v0.25.0.rst

```diff
@@ -269,7 +269,7 @@ Deprecations
 Removal of prior version deprecations/changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 - Removed ``Panel`` (:issue:`25047`, :issue:`25191`, :issue:`25231`)
--
+- Removed the previously deprecated ``ExcelWriter.sheetname`` (:issue:`16442`, :issue:`20938`)
```

Comment:
Actually is this part of `ExcelWriter` or `read_excel`? From the diff I think this should be changed to the latter

Reply:
Thanks for the catch! I shamelessly copied `ExcelWriter` from the deprecated issue (which was probably incorrect)